### PR TITLE
[Az.Resources]New features for MSGraph

### PR DESCRIPTION
**Pull request by VeryEarly**
## Description

- add update-azadgroup, 

- use beta api for groupmember, 

- support owned application, 

- expose additonalproperties for directoryobject

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Resources/MSGraph.Autorest/examples/Update-AzADGroup.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+### Example 1: {{ Add title here }}
```
Comment (isra-fel) [inline src/Resources/MSGraph.Autorest/examples/Update-AzADGroup.md:1]: Placeholder

**Review by isra-fel — CHANGES_REQUESTED** [review]
- Please suppress the breaking change issues and state why they are not valid.
- `Update-AzADGroup` returns nothing instead of the updated group. Please confirm.
Others LGTM. Thanks.

**Comment by VeryEarly** [comment]
> * Please suppress the breaking change issues and state why they are not valid.
> * `Update-AzADGroup` returns nothing instead of the updated group. Please confirm.
>   Others LGTM. Thanks.

Yes Patch operations for MSGraph do not have response body.

File: src/Resources/MSGraph.Autorest/examples/Get-AzADUserOwnedObject.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+### Example 1: {{ Add title here }}
```
Comment (isra-fel) [inline src/Resources/MSGraph.Autorest/examples/Get-AzADUserOwnedObject.md:1]: Placeholder

File: src/Resources/MSGraph.Autorest/docs/Az.MSGraph.md [hunk 3]
```diff
@@ -41,6 +41,12 @@ Lists key credentials and password credentials for an service principal.
 ### [Get-AzADUser](Get-AzADUser.md)
 Lists entities from users or get entity from users by key
 
+### [Get-AzADUserOwnedObject](Get-AzADUserOwnedObject.md)
```
Comment (dingmeng-xue) [inline src/Resources/MSGraph.Autorest/docs/Az.MSGraph.md:44]: Shall it be removed?

**Review by dingmeng-xue — APPROVED** [review]

**Review by isra-fel — APPROVED** [review]

**Pull request merged**